lib/suseconnect_register.pm

```diff
@@ -46,12 +46,14 @@ sub command_register {
     my $zypper_conflict = qr/^Choose from above solutions by number[\s\S,]* \[1/m;
     my $zypper_continue = qr/^Continue\? \[y/m;
     my $zypper_done     = qr/Run.*to list these programs|^ZYPPER-DONE/m;
+    my $registered      = qr/Registered*/m;
 
     if (!$addon) {
         my $reg_code = get_required_var("SCC_REGCODE");
         $version =~ s/\-SP/./;
         script_run("(SUSEConnect -p SLES/$version/$arch --regcode $reg_code) | tee /dev/$serialdev", 0);
-        my $out = wait_serial($zypper_conflict, 240);
+        my $out = wait_serial($zypper_conflict, 240) if (get_var("zdup"));    # zdup migration will have confilct
```

Comment:
I wonder if we could use command_register subroutine from suseconnect_register here, as there is already a mechanism to process conflicts.

Reply:
I have asked this pm's maintainer. this pm was wrote for zdup migration test case using , and also those conflict check.   if you want just register a system, it will not wait the expected serial output,please look the below link, so I modify this pm to wait another serial output to register successful. 
http://10.67.18.165/tests/250#step/smt_disconnect_prepare/31

Reply:
`get_var("zdup")` 'zdup' should be upper-letter?

Reply:
@mitiao modified, thanks!